src/status_im/infra/market/transform.cljs

```diff
@@ -0,0 +1,70 @@
+(ns status-im.infra.market.transform
+  (:require
+    [camel-snake-kebab.extras :as cske]
+    [utils.transforms :as transforms]))
+
+(defn json->clj-kebab-case
```

Comment:
Why do we need these general functions `json->clj-kebab-case` and `signal-payload-json->clj` in the market `transform` namespace? It looks like they should be reusable across different namespaces. For example, we could extend `utils.transforms/js->clj` to accept a second optional map for other transformations, such as kebab case.

Reply:
All makes sense, thanks @vkjr